mlrun/api/api/endpoints/background_tasks.py

```diff
@@ -12,19 +12,41 @@
     "/projects/{project}/background-tasks/{name}",
     response_model=mlrun.api.schemas.BackgroundTask,
 )
-def get_background_task(
+def get_project_background_task(
     project: str,
     name: str,
     auth_info: mlrun.api.schemas.AuthInfo = fastapi.Depends(
         mlrun.api.api.deps.authenticate_request
     ),
 ):
-    # Since there's no not-found option on get_background_task - we authorize before getting (unlike other get endpoint)
+    # Since there's no not-found option on get_project_background_task - we authorize before getting (unlike other
+    # get endpoint)
     mlrun.api.utils.auth.verifier.AuthVerifier().query_project_resource_permissions(
-        mlrun.api.schemas.AuthorizationResourceTypes.background_task,
+        mlrun.api.schemas.AuthorizationResourceTypes.project_background_task,
         project,
         name,
         mlrun.api.schemas.AuthorizationAction.read,
         auth_info,
     )
-    return mlrun.api.utils.background_tasks.Handler().get_background_task(project, name)
+    return mlrun.api.utils.background_tasks.Handler().get_project_background_task(
+        project, name
+    )
+
+
+@router.get(
+    "/background-tasks/{name}", response_model=mlrun.api.schemas.BackgroundTask,
+)
+def get_background_task(
+    name: str,
+    auth_info: mlrun.api.schemas.AuthInfo = fastapi.Depends(
+        mlrun.api.api.deps.authenticate_request
+    ),
+):
+    # Since there's no not-found option on get_background_task - we authorize before getting (unlike other get endpoint)
+    mlrun.api.utils.auth.verifier.AuthVerifier().query_resource_permissions(
```

Comment:
Adding this requires a change in the OPA manifest as well, adding this global endpoint to allowed users. Just noting this to make sure this is covered elsewhere.

Reply:
Good point - for 3.2 impossible, it's already out, for 3.4 possible but also not handled yet, skipping the checks for now